packages/framework-types/src/envelope.ts

```diff
@@ -68,14 +68,31 @@ export interface ReadModelEnvelope {
   value: ReadModelInterface
 }
 
+export interface ReadModelResult {
+  items: Array<unknown>
+  count?: number
+  cursor?: Record<string, string>
+}
+
 export interface ReadModelRequestEnvelope extends Envelope {
   typeName: string
   version: number
-  filters?: Record<string, ReadModelPropertyFilter>
+  filters?: ReadModelRequestProperties
```

Comment:
Not necessary in this PR, but FYI we found what it could be a bug with this type: https://github.com/boostercloud/booster/issues/879

Reply:
I'll take into account for future PRs as it requires some work